src/postfixCompletionProvider.ts

```diff
@@ -132,6 +132,21 @@ export class PostfixCompletionProvider implements vsc.CompletionItemProvider {
     return { currentNode: findNormalizedNode(source), fullSource, fullCurrentNode: findNormalizedNode(fullSource) }
   }
 
+  private convertToScriptKind(document: vsc.TextDocument) {
+    switch (document.languageId) {
+      case 'javascript':
+        return ts.ScriptKind.JS
+      case 'typescript':
+        return ts.ScriptKind.TS
+      case 'javascriptreact':
+        return ts.ScriptKind.JSX
+      case 'typescriptreact':
+        return ts.ScriptKind.TSX
+      default:
+        return ts.ScriptKind.Unknown

```

Comment:
Hmm, that means Vue/Svelte and HTML will have this unknown, have you checked if this still works?

Reply:
This is the same as `ts.ScriptKind.TS`

Reply:
All right!